packages/react/src/VideoPlayer/hooks/useVideoResizeObserver.ts

```diff
@@ -0,0 +1,46 @@
+import {type RefObject, useEffect, useState} from 'react'
+
+// TODO Once Safari 18 is released we can drop this in favour of container queries
+export const useVideoResizeObserver = ({
+  videoWrapperRef,
+  className,
+}: {
+  videoWrapperRef: RefObject<HTMLElement>
+  className: string
+}) => {
+  const [isSmall, setIsSmall] = useState(false)
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (!videoWrapperRef.current) return
+
+      const breakpoint = videoWrapperRef.current.getBoundingClientRect().width
+
+      const isSmallVideo = Boolean(breakpoint && breakpoint < 650)
```

Comment:
Just spotted this... our small breakpoint is actually `544`...  you can either pull in the constant from the tokens package or hardcode it, but shouldn't be 650 unless there's a good reason for it (let's leave comment).

<img width="346" alt="Screenshot 2024-07-18 at 10 49 05" src="https://github.com/user-attachments/assets/b8161e32-88e8-4316-8981-b96504d99ea4">

Reply:
This is definitely one that will benefit from a container query in the future as my guess is its going to be rare that videos are rendered at full width